lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -677,15 +677,20 @@ def install_a_puppet_msi_on(hosts, opts)
 
             if host.is_cygwin?
               # NOTE: it is critical that -o be before -O on Windows
-              on host, "curl -o \"#{msi_download_path}\" -O #{link}"
+              proxy = opts[:package_proxy] ? " -x #{opts[:package_proxy]}" : ''
+              on host, "curl#{proxy} -o \"#{msi_download_path}\" -O #{link}"
```

Comment:
I would move space to curl line for better readability
```
              proxy = opts[:package_proxy] ? "-x #{opts[:package_proxy]}" : ''
              on host, "curl #{proxy} -o \"#{msi_download_path}\" -O #{link}"

```

Reply:
I understand the concern, but this will create an unnecessary space in the event the package proxy isn't set. I can shift the leading space to the end of the the `proxy` variable string as an attempt to improve readability.